provider/cmd/pulumi-resource-eks/schema.json

```diff
@@ -1077,10 +1077,18 @@
                     "$ref": "/aws/v6.18.2/schema.json#/provider",
                     "description": "The AWS resource provider."
                 },
+                "clusterIngressRuleId": {
```

Comment:
The additions here can be copied into PR description to make searching easier. Referring to an internal doc is not that helpful for posterity. Thanks!

Reply:
Added it to the PR description